src/odemis/driver/smaract.py

```diff
@@ -1217,8 +1217,8 @@ def __init__(self, name, role, locator, axes, ref_on_init=False, linear_speed=0.
         # Connect to the device
         self._id = c_uint32(MC_5DOF_DLL.SA_MC_INVALID_HANDLE)
 
-        option_string = b"model " + str(MC_5DOF_DLL.hwModel) + "\n" + \
-                        b"locator " + locator
+        option_string = b"model " + str(MC_5DOF_DLL.hwModel).encode('ascii') + b"\n" + \
+                        b"locator " + locator.encode('ascii')
```

Comment:
I would suggest this:
```
        option_string = "model " + str(MC_5DOF_DLL.hwModel) + "\n" + \
                        "locator " + locator
                        
        options = c_char_p(option_string.encode("ascii"))
```

Reply:
Maybe this make it even more clear:
`option_string = "model %d\n locator %s" % (MC_5DOF_DLL.hwModel, locator)`